releases/release-4.0.9.md

```diff
@@ -0,0 +1,171 @@
+    - Fix the issue that the default value of newly added `Enum`-type columns are incorrect [#20998](https://github.com/pingcap/tidb/pull/20998)
+    - Fix the issue that the `addata` function inserts invalid characters [#21176](https://github.com/pingcap/tidb/pull/21176)
+    - Fix the issue that the wrong PointGet plan generated in some situations causes wrong results [#21244](https://github.com/pingcap/tidb/pull/21244)
+
++ TiKV
+
+    - Fix the issue that Coprocessor might return wrong results when there are more than 255 columns [#9131](https://github.com/tikv/tikv/pull/9131)
+    - Fix the issue that Region Merge might cause data loss during network partition [#9108](https://github.com/tikv/tikv/pull/9108)
+    - Fix the issue that the `ANALYZE` statement might cause panic when using the `latin1` character set [#9082](https://github.com/tikv/tikv/pull/9082)
+    - Fix the wrong results returned when converting the numeric type to the time type [#9031](https://github.com/tikv/tikv/pull/9031)
+    - Fix the bug that TiDB Lightning fails to ingest SST files to TiKV with the Importer-backend or Local-backend when
+ Transparent Data Encryption (TDE) is enabled [#8995](https://github.com/tikv/tikv/pull/8995)
+    - Fix the invalid `advertise-status-addr` value `0.0.0.0` [#9036](https://github.com/tikv/tikv/pull/9036)
+    - Fix the issue that an error is returned indicating that a key exists when this key is locked and deleted in a committed transaction [#8930](https://github.com/tikv/tikv/pull/8930)
+    - Fix the issue that the RocksDB mapping error causes data corruption [#9029](https://github.com/tikv/tikv/pull/9029)
+- Fix the bug that Follower Read might return stale data after the leader is transferred [#9240](https://github.com/tikv/tikv/pull/9240)
+
++ PD
+
+    - Fix the issue that the leader roles specified using placement rules do not take effect in some cases [#3208](https://github.com/pingcap/pd/pull/3208)
+    - Fix the issue that the `trace-region-flow` value is unexpectedly set to `false` [#3120](https://github.com/pingcap/pd/pull/3120)
+    - Fix a bug that the service safepoint with infinite Time To Live (TTL) does not work [#3143](https://github.com/pingcap/pd/pull/3143)
+
++ TiFlash
+
+    - Fix the issue that `INFORMATION_SCHEMA.CLUSTER_HARDWARE` might contain the information of disks that are not in use
+    - Fix the issue that the estimate on memory usage of Delta Cache is smaller than the actual usage
+    - Fix the memory leak caused by thread information statistics
+
++ Tools
+
+    + Backup & Restore (BR)
+
+        - Fix the failure caused by special characters in S3 secret access keys [#617](https://github.com/pingcap/br/pull/617)
+
+    + TiCDC
+
+        - Fix the issue that multiple owners might exist when the owner campaign key is deleted [#1104](https://github.com/pingcap/ticdc/pull/1104)
```

Comment:
```suggestion
        - Fix the issue that multiple owners might exist when the owner campaign key is deleted [#1104](https://github.com/pingcap/ticdc/pull/1104)
        - Fix a bug that TiCDC could fail to continue replicating when a TiKV crashes or recovers from a crash, the bug exists in v4.0.8 only [#1198](https://github.com/pingcap/ticdc/pull/1198)
         - Fix task status flushed to etcd too many before a table is initialized [#1191](https://github.com/pingcap/ticdc/pull/1191)
         - Fix schema storage may meet error because of update latency or GC too fast [#1114](https://github.com/pingcap/ticdc/pull/1114)
         - Fix schema storage costs too much memory when DDL is frequent [#1127](https://github.com/pingcap/ticdc/pull/1127)
         - Fix goroutine leak when changefeed is paused or stopped [#1075](https://github.com/pingcap/ticdc/pull/1075)
         - Increase the max retry timeout to 60s Kafka produce to prevent replication interruption because of service or network jitter in downstream Kafaka [#1118](https://github.com/pingcap/ticdc/pull/1118)
         - Fix the bug that Kafka batch size does not work [#1112](https://github.com/pingcap/ticdc/pull/1112)
```

Reply:
PTAL @TomShawn Many thanks

Reply:
No problem. Could you also provide the Chinese version in https://github.com/pingcap/docs-cn/pull/4976?